game/src/info/person.rs

```diff
@@ -191,9 +191,10 @@ pub fn trips(
             },
             {
                 // TODO Maybe generalize ImageSource::Bytes beyond just buttons
-                let mut icon = GeomBatch::from_uncached_svg_contents(include_bytes!(
-                    "../../../widgetry/icons/arrow_drop_down.svg"
-                ))
+                let mut icon = GeomBatch::load_svg_bytes(
```

Comment:
I was hasty with this fix over the weekend! Thanks for making it clear. At the time, I looked for other examples of caching, didn't immediately see any for non-buttons, and hence left that TODO.